src/Framework/QueryBuilder/Concerns/CRUD.php

```diff
@@ -29,6 +30,20 @@ public function insert($data, $format = null)
         );
     }
 
+    /**
+     * @unreleased
+     *
+     * @param array        $columns
+     * @param QueryBuilder $query
+     *
+     * @return bool|int
+     */
+    public function insertInto(array $columns, QueryBuilder $query)
```

Comment:
This is nice, but very limited. What if I want to pass values instead of the QB instance?

Reply:
This will be a great addition to QB, but let's leverage as many MySQL features as we can. Providing just a QB instance means that we can only use subqueries to select the data we want to insert, but I would like to do more than that. As I mentioned above, I would also like to have something like this:

```php
->insertInto(['name', 'last_name'], ['ante', 'laca']);
```

Or even something like this:

```php
->insertInto(['name', 'last_name'], [['ante', 'laca'], ['kyle', 'johnson']]);
```

Reply:
The biggest _problem_ with QB is that we are using `wpdb::insert`, `wpdb::update`, etc. under the hood in all CRUD methods.  Switching to _pure_ SQL will be much better, but that's something we can discuss some other time.

Reply:
I was looking into a dynamic second parameter, but union types require PHP 8.0, so I considered multiple methods (`insertDataInto`, `insertQueryInto`) and didn't like that. Still considering.

Reply:
@alaca I added a note to the PR explaining why I'm not including support for inserting multiple rows of known data as a part of this feature.